### ISSUE TITLE
LPCNet converge problem

### ISSUE THREAD

**Issue by bearlu007** [mozilla/LPCNet#36]
Hi all 
In my experiment for training LPCNet for TTS, it is found the LPCNet hard to converge and very like to generate local minimum . And the voice quality generated may vary in a large range . 
I have tried to reduce learning rate or change batch size or change training criterion, but these seems not helping. 
Do you have the same problem and any suggestions for training a stable LPCNet Vocoder quickly ?

Thanks
